plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -258,12 +259,20 @@ public ConnectorTableHandle getTableHandle(ConnectorSession session, SchemaTable
             columns.add(PARTITION_DATE.getColumnHandle());
             columns.add(PARTITION_TIME.getColumnHandle());
         }
+        Optional<String> connectionId = Optional.empty();
+        if (tableInfo.get().getDefinition() instanceof ExternalTableDefinition) {
+            ExternalTableDefinition edt = tableInfo.get().getDefinition();
```

Comment:
Avoid abbreviations. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-abbreviations

Also, you can use enhanced instanceof to define this variable after ExternalTableDefinition.

Reply:
Done.